src/matrix/inversewishart.jl

```diff
@@ -43,6 +43,16 @@ size(d::InverseWishart) = (p = dim(d); (p, p))
 params(d::InverseWishart) = (d.df, d.Ψ, d.c0)
 @inline partype{T<:Real}(d::InverseWishart{T}) = T
 
+### Conversion
+function convert{T<:Real}(::Type{InverseWishart{T}}, d::InverseWishart)
+    P = convert_eltype(T, d.Ψ)
```

Comment:
The way it's done in Base is that each special matrix defines
```jl
convert(::Type{AbstractMatrix{T},A::SpecialMatrix) = convert(SpecialMatrix{T}, A)
```
However, your second comment is quite relevant. We might want to decouple conversion of the element type from abstract types. Many different possibly unrelated types could have elements for which it would be relevant to convert them. So it's worth considering if Base should adopt something like `convert_eltype` instead of opposite.